### ISSUE TITLE
Chrome version download error in dockerfile template

### ISSUE THREAD

**Issue by OKT96** [odoo/runbot#732]
The chrome_version 90.0.4430.93-1 specified in the dockerfile template is unable to be downloaded due to a 404 error and causes the docker build process to fail

```bash
Step 15/18 : RUN curl -sSL https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_90.0.4430.93-1_amd64.deb -o /tmp/chrome.deb     && apt-get update     && apt-get -y install --no-install-recommends /tmp/chrome.deb     && rm /tmp/chrome.deb
 ---> Running in 6e90e215cfa8
Get:1 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]
Get:2 https://deb.nodesource.com/node_15.x focal InRelease [4583 B]
Get:3 http://archive.ubuntu.com/ubuntu focal InRelease [265 kB]
Get:4 https://deb.nodesource.com/node_15.x focal/main amd64 Packages [770 B]
Get:5 http://apt.postgresql.org/pub/repos/apt focal-pgdg InRelease [91.6 kB]
Get:6 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [2448 kB]
Get:7 http://security.ubuntu.com/ubuntu focal-security/restricted amd64 Packages [1882 kB]
Get:8 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [990 kB]
Get:9 http://security.ubuntu.com/ubuntu focal-security/multiverse amd64 Packages [27.7 kB]
Get:10 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
Get:11 http://archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]
Get:12 http://apt.postgresql.org/pub/repos/apt focal-pgdg/main amd64 Packages [419 kB]
Get:13 http://archive.ubuntu.com/ubuntu focal/multiverse amd64 Packages [177 kB]
Get:14 http://archive.ubuntu.com/ubuntu focal/restricted amd64 Packages [33.4 kB]
Get:15 http://archive.ubuntu.com/ubuntu focal/main amd64 Packages [1275 kB]
Get:16 http://archive.ubuntu.com/ubuntu focal/universe amd64 Packages [11.3 MB]
Get:17 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [2921 kB]
Get:18 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [31.2 kB]
Get:19 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1291 kB]
Get:20 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [2009 kB]
Get:21 http://archive.ubuntu.com/ubuntu focal-backports/main amd64 Packages [55.2 kB]
Get:22 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [28.6 kB]
Fetched 25.6 MB in 5s (5562 kB/s)
Reading package lists...
Reading package lists...
E: Invalid archive signature
E: Internal error, could not locate member control.tar.{zstlz4gzxzbz2lzma}
E: Could not read meta data from /tmp/chrome.deb
E: The package lists or status file could not be parsed or opened.
The command '/bin/sh -c curl -sSL https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_90.0.4430.93-1_amd64.deb -o /tmp/chrome.deb     && apt-get update     && apt-get -y install --no-install-recommends /tmp/chrome.deb     && rm /tmp/chrome.deb' returned a non-zero code: 100
```

**Comment by d-fence** [comment]
Hi. The Dockerfile template provided in the code is just a base that have to be overridden with `chrome_version`
This version is indeed outdated.
Actually we prefer to fix the version to keep consistency accros runbot's when the runbot is used on multiple hosts.

**Comment by OKT96** [comment]
Hi @d-fence thanks for your reply. We think that using the latest version for chrome could lead to other issues. 
Do you have recommendations on which chrome version to use for Odoo 15?
Thanks

**Comment by Xavier-Do** [comment]
> Hi @d-fence thanks for your reply. We think that using the latest version for chrome could lead to other issues. Do you have recommendations on which chrome version to use for Odoo 15? Thanks

For testing, it depends of you needs. Changing chrome version can introduce new errors.
- less deterministic behaviour
- memory consumption
- es version
- security policies
- flags
- ...

On runbot.odoo.com we need to ensure that tests are consistent cross hosts and also that an update doesn't suddenly break tests. This is why the version is freezes.

For your own runbot, as far as you tests are green, use whatever version you want. The version we are using works fine for  odoo tests at the moment.

We usually upgrade the version when:
- a new feature is needed
- the version is old and hides a real bug that we may have in the latest chrome release 
- the version is too old

The current version is fixed to 101.0.4951.64-1

A new chrome version needs to be tested on the complete code base before applying it in the default docker images. 
We have a nightly build for that there: https://runbot.odoo.com/runbot/build/23487102 but we currently have an error (version 107.0.5304.87)

Following this issue, we will update the docker image of this build with version 109.0.5414.119 to check what else is breaking (or if it fixes the previous issue)
